### PR TITLE
Fix/readme

### PR DESCRIPTION
**Pull request by alfjim009**
**What this PR does / why it's submitted / why we need it**:
- Updated readme files
- Added Cache Header View component name constant
- Updated UI with new changes

---

**Checklist**:

- [ ] My PR has a descriptive title (not a vague title like `Needed this`)
- [ ] Title and all the commits follow [commitlint guidelines](https://github.com/conventional-changelog/commitlint#what-is-commitlint)
- [ ] My PR targets the `develop` branch or appropriate `release` branch
- [ ] My PR provides test coverage for my change (where applicable)
- [ ] I've placed the link to the corresponding Trello card or issue which this PR fixes/address below ⤵️

---

Fixes: ### ISSUE_LINK ###

### CODE REVIEW

File: src/Cogworks.CogFlare.Core/README.md [hunk 1]
```diff
@@ -11,8 +11,8 @@ A package that helps automatically purge CloudFlare cache
 *   When any content node is changed(**Published/Unpublished/Deleted**), that node and any nodes referencing it will be purged from CloudFlare's cache
 *   When any media item is changed(**Saved**), it will be purged from CloudFlare's cache
 *   The ability to toggle the package functionality on/off in the settings
-*   The ability to toggle the cloudfarte cache by adding the property "disableCloudFlareCache" to the node you wish to ignore
-*   If you want to bypass a page now you have 2 options, using the "disableCloudFlareCache" in the node or using the "BlockListPropertyAlias" and "BlockAliases" in the settings
+*   The ability to toggle the cloudfare cache by adding the property "disableCloudFlareCache" to the node you wish to ignore
```
Comment (michalszczepanikcogworks) [inline src/Cogworks.CogFlare.Core/README.md]: Cloudflare

File: src/Cogworks.CogFlare.Core/README.md [hunk 2]
```diff
@@ -52,12 +52,30 @@ Add these settings to the **appsettings.json**
...
 ```
 
+For bypass logic you can use 2 options :
+1. Adding a new property "disableCloudFlareCache" in the node where you want to disable cache :
+![DisableCloudFare-Property](./disableCloudFareCacheProperty.jpg)
+2. Adding the block aliases which will be included in the view where you want to disable cache :
+    ```js
+      "CogFlareSettings": {
+        ...
+        "BlockAliases": "formBlock,newBlock, ...",
```
Comment (michalszczepanikcogworks) [inline src/Cogworks.CogFlare.Core/README.md]: Add a space after the "formBlock," please

File: src/Cogworks.CogFlare.Core/README.md [hunk 3]
```diff
@@ -52,12 +52,30 @@ Add these settings to the **appsettings.json**
...
     "KeyNodes": "1234, 031089",
     "IsEnabled": true,
-    "BlockListPropertyAlias": "",
     "BlockAliases": "formBlock,newBlock",
     "Domain": "https://www.example.com"
   }
 ```
 
+For bypass logic you can use 2 options :
+1. Adding a new property "disableCloudFlareCache" in the node where you want to disable cache :
```
Comment (michalszczepanikcogworks) [inline src/Cogworks.CogFlare.Core/README.md]: Remove space before the : please (unless it's some special md character I don't know about)

File: src/Cogworks.CogFlare.UI/Views/HomePage.cshtml [hunk 4]
```diff
@@ -1,4 +1,6 @@
-﻿@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<Cogworks.CogFlare.UI.Models.CmsModels.Generated.HomePage>
+﻿@using Cogworks.CogFlare.Core.Constants
+@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<Cogworks.CogFlare.UI.Models.CmsModels.Generated.HomePage>
 @{
 	Layout = null;
+    @await Component.InvokeAsync(ApplicationConstants.CogFlareCacheHeaders)
```
Comment (michalszczepanikcogworks) [inline src/Cogworks.CogFlare.UI/Views/HomePage.cshtml:5]: Format this please

File: README.md [hunk 5]
```diff
@@ -13,8 +13,8 @@ A package that helps automatically purge CloudFlare cache
 *   When any content node is changed(**Published/Unpublished/Deleted**), that node and any nodes referencing it will be purged from CloudFlare's cache
 *   When any media item is changed(**Saved**), it will be purged from CloudFlare's cache
 *   The ability to toggle the package functionality on/off in the settings
-*   The ability to toggle the cloudfarte cache by adding the property "disableCloudFlareCache" to the node you wish to ignore
-*   If you want to bypass a form page now you have 2 options, using the "disableCloudFlareCache" in the form node or using the "BlockListPropertyAlias" and "FormBlockAlias" in the settings
+*   The ability to toggle the cloudflare cache by adding the property "disableCloudFlareCache" to the node you wish to ignore
```
Comment (michalszczepanikcogworks) [inline README.md]: Cloudflare

File: src/Cogworks.CogFlare.Core/README.md [hunk 6]
```diff
@@ -11,8 +11,8 @@ A package that helps automatically purge CloudFlare cache
 *   When any content node is changed(**Published/Unpublished/Deleted**), that node and any nodes referencing it will be purged from CloudFlare's cache
 *   When any media item is changed(**Saved**), it will be purged from CloudFlare's cache
 *   The ability to toggle the package functionality on/off in the settings
-*   The ability to toggle the cloudfarte cache by adding the property "disableCloudFlareCache" to the node you wish to ignore
-*   If you want to bypass a page now you have 2 options, using the "disableCloudFlareCache" in the node or using the "BlockListPropertyAlias" and "BlockAliases" in the settings
+*   The ability to toggle the cloudflare cache by adding the property "disableCloudFlareCache" to the node you wish to ignore
```
Comment (michalszczepanikcogworks) [inline src/Cogworks.CogFlare.Core/README.md]: Cloudflare

File: src/Cogworks.CogFlare.Core/README.md [hunk 7]
```diff
@@ -52,12 +52,30 @@ Add these settings to the **appsettings.json**
...
+      "CogFlareSettings": {
+        ...
+        "BlockAliases": "formBlock, newBlock, ...",
+        ...
+      }
+    ```
+    And then you need to add the view component in your Master or required page View
+    ```razor
+      @using Cogworks.CogFlare.Core.Constants
+      @{
```
Comment (michalszczepanikcogworks) [inline src/Cogworks.CogFlare.Core/README.md]: @{	
            @await Component.InvokeAsync(ApplicationConstants.CogFlareCacheHeaders)
      }

File: src/Cogworks.CogFlare/README.md [hunk 8]
```diff
@@ -50,10 +52,30 @@ Add these settings to the **appsettings.json**
...
+      "CogFlareSettings": {
+        ...
+        "BlockAliases": "formBlock, newBlock, ...",
+        ...
+      }
+    ```
+    And then you need to add the view component in your Master or required page View
+    ```razor
+      @using Cogworks.CogFlare.Core.Constants
+      @{
```
Comment (michalszczepanikcogworks) [inline src/Cogworks.CogFlare/README.md]: @{	
            @await Component.InvokeAsync(ApplicationConstants.CogFlareCacheHeaders)
      }

File: README.md [hunk 9]
```diff
@@ -58,12 +58,30 @@ Add these settings to the **appsettings.json**
...
+      "CogFlareSettings": {
+        ...
+        "BlockAliases": "formBlock, newBlock, ...",
+        ...
+      }
+    ```
+    And then you need to add the view component in your Master or required page View
+    ```razor
+      @using Cogworks.CogFlare.Core.Constants
+      @{
```
Comment (michalszczepanikcogworks) [inline README.md:79]: @{	
            @await Component.InvokeAsync(ApplicationConstants.CogFlareCacheHeaders)
      }

**Comment by michalszczepanikcogworks** [comment]
Why is there so many READMEs?

**Comment by alfjim009** [comment]
Uhmm 1 it's from the solution, and the other 2 from each project. But the 3 are the same, I just updated the existing files and I think we should keep it, to have the info in each nuget package page @michalszczepanikcogworks .

**Review by michalszczepanikcogworks — APPROVED** [review]

**Pull request merged**